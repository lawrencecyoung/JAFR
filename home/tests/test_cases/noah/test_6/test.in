1
0
16
0 1 2
a
1a
1.
5 6
1
10 11 12 13 14
10 11 12 13
6


