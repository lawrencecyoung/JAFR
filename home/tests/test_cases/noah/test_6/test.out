Just a friendly reminder! You have these tasks to finish today.
- Has indentation before
- 
- Dash-in-Description
- Missing space character
- Duplicates should show up twice
- Duplicates should show up twice
- Special characters !@#$%^&*()_+{}|~:"<>?
- D/M/YY
- Not complete task should show up
- Task due today

These tasks need to be finished in the next three days!
- Task due tomorrow by 28/08/23
- Task due in two days by 29/08/23
- Task due in three days by 30/08/23

You have the following meetings today!
- Has indentation before at 18:00
-  at 18:00
- Dash-in-Description at 18:00
- Missing space character at 18:00
- Duplicates should show up twice at 18:00
- Duplicates should show up twice at 18:00
- Special characters !@#$%^&*()_+{}|~:"<>? at 18:00
- Edge Valid hour value at 23:59
- Edge Valid hour value at 00:00
- Edge Valid Minute value at 13:00
- Edge Valid Minute value at 13:59
- Meeting due today at 18:00

You have the following meetings scheduled over the next week!
- Meeting due tomorrow on 28/08/23 at 18:00
- Meeting due in two days on 29/08/23 at 18:00
- Meeting due in three days on 30/08/23 at 18:00
- Meeting due in four days on 31/08/23 at 18:00
- Meeting due in five days on 01/09/23 at 18:00
- Meeting due in six days on 02/09/23 at 18:00
- Meeting due in the next week on 03/09/23 at 18:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Which task(s) would you like to mark as completed?
1. Has indentation before by 27/08/23
2.  by 27/08/23
3. Dash-in-Description by 27/08/23
4. Missing space character by 27/08/23
5. Duplicates should show up twice by 27/08/23
6. Duplicates should show up twice by 27/08/23
7. Special characters !@#$%^&*()_+{}|~:"<>? by 27/08/23
8. D/M/YY by 27/8/23
9. Not complete task should show up by 27/08/23
10. Task due yesterday by 26/08/23
11. Task due today by 27/08/23
12. Task due tomorrow by 28/08/23
13. Task due in two days by 29/08/23
14. Task due in three days by 30/08/23
15. Task due in four days by 31/08/23
Invalid choice of tasks
Invalid choice of tasks
Invalid choice of tasks
Invalid choice of tasks
Invalid choice of tasks
Invalid choice of tasks
Marked as complete.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Which task(s) would you like to mark as completed?
1. Has indentation before by 27/08/23
2.  by 27/08/23
3. Dash-in-Description by 27/08/23
4. Missing space character by 27/08/23
5. Special characters !@#$%^&*()_+{}|~:"<>? by 27/08/23
6. D/M/YY by 27/8/23
7. Not complete task should show up by 27/08/23
8. Task due yesterday by 26/08/23
9. Task due today by 27/08/23
10. Task due tomorrow by 28/08/23
11. Task due in two days by 29/08/23
12. Task due in three days by 30/08/23
13. Task due in four days by 31/08/23
Invalid choice of tasks
Marked as complete.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
